dbt/include/databricks/macros/adapters.sql

```diff
@@ -111,6 +111,37 @@
   {% endif %}
 {% endmacro %}
 
+{% macro optimize(relation) %}
+  {{ return(adapter.dispatch('optimize', 'dbt')(relation)) }}
+{% endmacro %}
+
+{% macro databricks__optimize(relation) %}
+  {% if config.get('zorder', False) and config.get('file_format', 'delta') == 'delta' %}
```

Comment:
Again I'm unfamiliar with how much of Jinja2 dbt uses, but customarily the `'delta'` string literal here would be a defined constant that's available to Jinja's context, so we can change it down the line. If that's not possible then this line is good.

Reply:
delta is specific to our adapter. Not something that can come from jinja. The context is created by dbt-core.